index.html

```diff
@@ -37,6 +36,146 @@ <h4>LET’S CONNECT</h4>
                  </ul>
             </div>
     </section>
+
+    <section class="works">
+        <div class="cards">
+       <div class="card-1">
+          <img src="images/Snapshoot-Portfolio.jpg" alt="">
+         
+           <h1>Tonic</h1>  
```

Comment:
- [ ] There should be only one `h1` in an HTML file because of best practices for SEO. I kindly suggest that you make only your main title of "I’m Zelalem Glad to see you!" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.
- [ ] please kindly make changes and replace all other `h1` except your main title